rules/network/command_and_control_rdp_remote_desktop_protocol_from_the_internet.toml

```
@@ -33,7 +33,7 @@ tags = ["Elastic", "Network"]
 type = "query"
 
 query = '''
-network.transport:tcp and destination.port:3389 and
+event.category:(network or network_traffic) and network.transport:tcp and (destination.port:3389 or event.dataset:zeek.rdp or (event.dataset:zeek.ssl and destination.port:3389)) and
```

technically `event.dataset:zeek.ssl` will never hit since it requires 3389 which already exists in higher precedence.
```suggestion
event.category:(network or network_traffic) and network.transport:tcp and (destination.port:3389 or event.dataset:zeek.rdp) and
```